cypress/e2e/onboarding.ts

```diff
@@ -30,11 +30,13 @@ describe('onboarding', () => {
     cy.findByRole('main').within(() => {
       cy.findByRole('textbox', {name: /name/i}).type(firstName)
       cy.findByRole('group', {name: /team/i}).within(() => {
-        cy.findByRole('radio', {name: /blue/i}).click()
+        // checkbox is covered with a <label>
+        cy.findByRole('radio', {name: /blue/i}).click({force: true})
       })
-      cy.findByRole('button', {name: /join KCD/i}).click()
+      cy.findByRole('button', {name: /create account/i}).click()
 
-      cy.findByRole('button', {name: /logout/i}).click()
+      // TODO: find a new home for the logout button?
+      // cy.findByRole('button', {name: /logout/i}).click()
```

Comment:
Ok. This is not so awesome. But the current designs haven't taken them into account, and I lack the creativity to come up with something nice. I'll ask Gil.

Reply:
Ok. Gonna try something while I wait for the designer. Got some inspiration.

Reply:
Done.

![image](https://user-images.githubusercontent.com/1196524/124292011-74032b80-db55-11eb-9988-04c09fffbe10.png)